src/controllers/dlgprefcontrollers.cpp

```diff
@@ -19,24 +19,21 @@ DlgPrefControllers::DlgPrefControllers(DlgPreferences* pPreferences,
     setupUi(this);
     setupControllerWidgets();
 
-    connect(&m_buttonMapper, SIGNAL(mapped(QString)),
-            this, SLOT(slotOpenLocalFile(QString)));
-
-    connect(btnOpenUserPresets, SIGNAL(clicked()),
-            &m_buttonMapper, SLOT(map()));
-
-    m_buttonMapper.setMapping(btnOpenUserPresets, userPresetsPath(m_pConfig));
+    connect(btnOpenUserPresets, &QPushButton::clicked, [=]() {
+            QString presetsPath = userPresetsPath(m_pConfig);
+            openLocalFile(presetsPath);
+    });
```

Comment:
Removing the QSignalMapper is okay in this case. If you need to use QSignalMapper with the new syntax in the future, there's a [workaround](http://doc.qt.io/qt-5/qsignalmapper.html#mapped) you need to use.